tests/serialization_tests/tests/signing_ser.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+// Doesn't run these unless feature specified
+#![cfg(feature = "submodule_tests")]
+#![allow(dead_code, non_snake_case)]
+
+use bls_signatures::{PrivateKey, Serialize};
+use crypto::{signature, Signature};
+use encoding::Cbor;
+use forest_message::{unsigned_message, UnsignedMessage};
+use serde::Deserialize;
+use std::fs::File;
+use std::io::prelude::*;
+
+#[derive(Deserialize)]
+struct TestVec {
+    #[serde(with = "unsigned_message::json")]
+    Unsigned: UnsignedMessage,
+    Cid: String,
+    CidHexBytes: String,
+    PrivateKey: String,
+    #[serde(with = "signature::json")]
+    Signature: Signature,
+}
```

Comment:
```suggestion
#[derive(Deserialize)]
#[serde(rename_all = "PascalCase")]
struct TestVec {
    #[serde(with = "unsigned_message::json")]
    unsigned: UnsignedMessage,
    cid: String,
    private_key: String,
    #[serde(with = "signature::json")]
    signature: Signature,
}
```
idc because it's a test but you don't need to allow non snake case to do this. (Also you don't need to include the cid hex bytes and idk why they have this)